scout/utils/convert.py

```diff
@@ -17,6 +17,14 @@ def isint(x):
         return a == b
 
 
+def make_bool_pass_none(string):
+    if string is None:
```

Comment:
It works but perhaps if you are sure that the returned value if "None" or else then you could just write:
if string is None or str == 'None:

Reply:
Now that you mention it, the str()-cast is None safe, so the first isn't needed at all! Thanks!